transport/src/main/java/io/netty/channel/ChannelHandlerMask.java

```diff
@@ -163,7 +167,14 @@ private static boolean isSkippable(
         return AccessController.doPrivileged(new PrivilegedExceptionAction<Boolean>() {
             @Override
             public Boolean run() throws Exception {
-                return handlerType.getMethod(methodName, paramTypes).isAnnotationPresent(Skip.class);
+                Method m;
+                try {
+                    m = handlerType.getMethod(methodName, paramTypes);
+                } catch (NoSuchMethodException e) {
+                    logger.debug("Class {} missing method {}.", handlerType, methodName, e);
```

Comment:
`e` is being thrown away. Just do `logger.debug(e)` instead? (Yes, it has a bit less data, but I don't expect the extra data to help too much.)

Reply:
I think it would be good to log a bit more ... But maybe we should make the log line a big more clear:

`logger.debug("Class {} missing method {}, assume we can not skip execution", handlerType, methodName, e);`

Reply:
I can add more detail, but I thought I recall trailing throwables being treated special by the logger.

Reply:
Yes they are treated special...